department-of-veterans-affairs/vets-website
# [Disability Calculator] Apply focus to results after submit button is clicked

## Description
This PR applies focus to the results container after the form is submitted. 

Ticket - https://github.com/department-of-veterans-affairs/va.gov-team/issues/431

## Testing done
1. Add a rating
2. Add a rating
3. Click `Calculate`
4. Verify focus

I repeated this same test while adding and removing ratings.

## Screenshots
N/A

## Acceptance criteria
- [ ] Focus is applied on results container after submit

## Definition of done
